code/src/components/Header.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import "./form.css";
+
+const Header = ({ title, subtitle }) => {
+  return (
+    <section className="section-container">
+      <h1 tabIndex="0">{title}</h1>
```

Comment:
Do you need the tabIndex here? I think the H1 is enough for accessibility unless you can interact with the title